WordPress/src/main/java/org/wordpress/android/viewmodel/ContextProvider.kt

```diff
@@ -0,0 +1,18 @@
+package org.wordpress.android.viewmodel
+
+import android.content.Context
+import org.wordpress.android.util.LocaleManager
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class ContextProvider
+@Inject constructor(private var context: Context) {
+    fun refreshContext() {
+        this.context = LocaleManager.setLocale(this.context)
+    }
+
+    fun getContext(): Context {
+        return context
+    }
```

Comment:
Really minor np: what about using expression body function here? Eventually keeping the return data type for readability. I'm in any case fine with as it is already.

Reply:
sounds good 👍 anything that reduces the boilerplate code is a good change :-)